ena-submission/scripts/test_ena_submission_integration.py

```diff
@@ -445,15 +446,82 @@ def simple_submission(
     mock_submit_external_metadata.return_value = mock_requests_post()
     sequences_to_upload: Final = get_sequences()
 
+    get_external_metadata_and_send_to_loculus(db_config, config)
+    mock_submit_external_metadata.assert_not_called()
+
     upload_sequences(db_config, sequences_to_upload)
     check_sequences_uploaded(db_config, sequences_to_upload)
+    get_external_metadata_and_send_to_loculus(db_config, config)
+    mock_submit_external_metadata.assert_not_called()
 
     _test_successful_project_submission(db_config, config, sequences_to_upload)
+    get_external_metadata_and_send_to_loculus(db_config, config)
+    args = mock_submit_external_metadata.call_args_list
+
+    assert len(args) == 1
+    payload = args[0][0][0]  # first positional argument of first call
+    assert payload["accession"] == "LOC_0001TLY"
+    assert payload["version"] == 1
+    assert set(payload["externalMetadata"]) == {"bioprojectAccession"}
+    assert payload["externalMetadata"]["bioprojectAccession"].startswith("PRJEB")
+    assert mock_submit_external_metadata.call_args[0][0]["accession"] == "LOC_0001TLY"
+    assert mock_submit_external_metadata.call_args[0][0]["version"] == 1
```

Comment:
isn't this a repeat of:
assert payload["accession"] == "LOC_0001TLY"
assert payload["version"] == 1
```suggestion
```

Reply:
indeed